embed/Makefile.am

```diff
@@ -43,6 +43,10 @@ libephyembed_la_SOURCES = \
 	ephy-file-monitor.h		\
 	ephy-find-toolbar.c		\
 	ephy-find-toolbar.h		\
+	ephy-fx-password-notification.c	\
```

Comment:
fx? (Special effects?) Consider naming the file simply `ephy-password-notification`